src/PlayerbotAI.cpp

```diff
@@ -4137,16 +4139,16 @@ bool PlayerbotAI::AllowActive(ActivityType activityType)
 {
     // only keep updating till initializing time has completed,
     // which prevents unneeded expensive GameTime calls.
-    if (_isBotInitializing)
-    {
-        _isBotInitializing = GameTime::GetUptime().count() < sPlayerbotAIConfig->maxRandomBots * 0.12;
+    // if (_isBotInitializing)
```

Comment:
not sure, it shouldnt be disabled but maybe there is a reason for it? Only @liyunfan1223 can answer that

Reply:
It's my fault that I didn't explain this change. I commented it because rndbots will not fight back when attacked by mobs during the initialization, so they will die a lot. I didn't experienced significant negative impact on performance on this modification.

This is just my opinion, do you think it needs to be kept?

Reply:
True thats a side effect. Its alright to leave it off, maybe i can optimilaze. Need to think about it. Thanks for replying.